appcues/src/main/java/com/appcues/ui/extensions/ModifierExt.kt

```diff
@@ -126,8 +117,13 @@ internal fun Modifier.styleBorder(
     }
 )
 
-internal fun Modifier.styleSize(style: ComponentStyle, contentMode: ComponentContentMode = ComponentContentMode.FIT) = this.then(
+internal fun Modifier.styleSize(
+    style: ComponentStyle,
+    matchParentBox: BoxScope? = null,
+    contentMode: ComponentContentMode = ComponentContentMode.FIT,
+) = this.then(
     when {
+        matchParentBox != null -> with(matchParentBox) { Modifier.matchParentSize() }
```

Comment:
matchParentBox takes precedent over anything else when its not null.

so if for some reason we would like to build a background where the image is not fullscreen, we would need to put that into a stack and then size it (width/height).

this is in parity with IOS afaik